Signal/src/ViewControllers/RegistrationViewController.m

```diff
@@ -48,6 +48,16 @@ - (void)viewDidLoad {
     [_sendCodeButton setTitle:NSLocalizedString(@"REGISTRATION_VERIFY_DEVICE", @"") forState:UIControlStateNormal];
     [_existingUserButton setTitle:NSLocalizedString(@"ALREADY_HAVE_ACCOUNT_BUTTON", @"registration button text")
                          forState:UIControlStateNormal];
+
+    [self.countryNameButton addTarget:self
+                               action:@selector(changeCountryCodeTapped)
+                     forControlEvents:UIControlEventTouchUpInside];
+    [self.countryCodeButton addTarget:self
+                               action:@selector(changeCountryCodeTapped)
+                     forControlEvents:UIControlEventTouchUpInside];
+    [self.countryCodeRow
+        addGestureRecognizer:[[UITapGestureRecognizer alloc] initWithTarget:self
+                                                                     action:@selector(countryCodeRowWasTapped:)]];
```

Comment:
I made it easier it to activate the country code view by accepting taps anywhere in the country code row.